cloud-editor/src/EditorButtonControl.js

```diff
@@ -51,17 +38,13 @@ export class EditorButtonControl extends AppComponent {
     });
 
     this.sub('on.click', (onClick) => {
-      setAriaClick(this, onClick);
+      this.onclick = onClick;
     });
   }
 }
 
-EditorButtonControl.renderShadow = false;
-
 EditorButtonControl.template = /*html*/ `
   <div class="before"></div>
-  <div class="icon" set="innerHTML: tpl.icon"></div>
+  <uc-icon size="20" set="@name: icon;"></uc-icon>
   <div class="title" ref="title-el" set="textContent: title"></div>
```

Comment:
Maybe, It would be more consistent to use text node bindings `{{txt}}` instead of `textContent` fields everywhere... I think it's little bit slower, but more aesthetic for code discovery... Anyway, we can discuss that one more time.